pokemongo_bot/__init__.py

```diff
@@ -39,12 +43,31 @@ def __init__(self, config):
         self.latest_inventory = None
         self.cell = None
 
+
     def start(self):
         self._setup_logging()
         self._setup_api()
         self.navigator = SpiralNavigator(self)
         random.seed()
 
+    def _setup_event_system(self):
```

Comment:
Why does this belong here and not in the pokecli.py file? If it was in the cli file we'd be able to send an event on startup with the bag information we are currently printing to the console, right?